tpl/template_funcs.go

```diff
@@ -330,6 +330,9 @@ func In(l interface{}, v interface{}) bool {
 	case reflect.Array, reflect.Slice:
 		for i := 0; i < lv.Len(); i++ {
 			lvv := lv.Index(i)
+			if lvv.Kind() == reflect.Interface {
+				lvv = reflect.ValueOf(lvv.Interface())
```

Comment:
This line

```go
lvv = reflect.ValueOf(lvv.Interface())
```

can be replaced by

```go
lvv = lvv.Elem()
```

and it should be better to use

```go
lvv, isNil := indirect(lvv)
if isNil {
	continue // or something
}
```

idiom instead of

```go
if lvv.Kind() == reflect.Interface {
	lvv = reflect.ValueOf(lvv.Interface())
}
```